scripts/ssb-web-pi/ssb-web-pi.conf

```diff
@@ -1,12 +1,12 @@
-location /sbot {
+location /ssb-web-pi {
     alias /var/www/sbot;
```

Comment:
Is this alias still valid?

Reply:
I didn't want to change the actual code due to lost of hard coded things in the code stack. Module is very ALPHA still but has some interesting concepts.

Didnt want to break things.